packages/cli/lib/projectsWatch.js

```diff
@@ -12,11 +12,12 @@ const { isAllowedExtension } = require('@hubspot/local-dev-lib/path');
 const { shouldIgnoreFile } = require('@hubspot/local-dev-lib/ignoreRules');
 const {
   cancelStagedBuild,
+  //TODO LDL does not export these
```

Comment:
Do we need to figure this out before merging?

Reply:
Thanks for catching this. I think I need to get this stuff ported to LDL before we can merge. I had forgotten about this.

Reply:
Ported [here](https://github.com/HubSpot/hubspot-local-dev-lib/pull/104), but still needs testing. I'll do that tomorrow